### PR TITLE
Upgrade to CircleCI 2.0

### PR DESCRIPTION
**Pull request by mikenikles**
Apply changes to run on CircleCI 2.0. Please see the linked PR for details.

 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/425

## How to test-drive this PR
The test instructions are the same as in the linked SDK PR.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+      - type: shell
+        name: Install Dependencies
+        command: npm install
+      - type: cache-save
+        key: progressive-web-scaffold-{{ .Branch }}
+        paths:
+          - /home/ubuntu/progressive-web-scaffold/node_modules
+      - type: shell
+        name: NPM Test
+        command: npm run lint && npm test -- --coverage
```
Comment (kschmidtdev) [inline circle.yml]: This could just be `npm run test:all`

File: circle.yml [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+      - type: shell
+        name: Install Chrome
+        command: |
+          wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+          sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+          apt-get update
+          apt-get --only-upgrade install google-chrome-stable
+      - type: shell
+        name: Install Dependencies
+        command: npm install
```
Comment (kschmidtdev) [inline circle.yml]: Thoughts on making this `npm i --no-spin --no-progress` ? I'm not sure if `npm` fixed their performance issues with showing a progress bar and spinner yet.

Comment (mikenikles) [inline circle.yml]: It looks like `npm` 3.x doesn't have that issue anymore, based on https://github.com/npm/npm/issues/11283#issuecomment-251924063.

File: circle.yml (hunk 1 above)
Comment (mikenikles) [inline circle.yml]: Good point. We recently added `--coverage` to that command. Change incoming.

**Comment by mikenikles** [comment]
Thanks for the reviews guys!

**Pull request merged**